nautobot/users/admin.py

```diff
@@ -241,7 +241,8 @@ def clean(self):
             for ct in object_types:
                 model = ct.model_class()
                 try:
-                    model.objects.filter(*[models.Q(**c) for c in constraints]).exists()
+                    tokens = {"$user": None}  # setting token to null user ID
+                    model.objects.filter(qs_filter_from_constraints(constraints, tokens)).exists()
```

Comment:
Just confirming - have you tested this out in the admin UI to make sure it works as intended?

Reply:
Hey yes! I didn't notice any weird issues